packages/utils/rollup.config.ts

```diff
@@ -24,7 +24,7 @@ export default [
             { file: pkg.exports["."].node.require, format: "cjs", banner, exports: "auto" },
             { file: pkg.exports["."].node.default, format: "es", banner }
         ],
-        external: [...Object.keys(pkg.dependencies), "crypto"],
+        external: [...Object.keys(pkg.dependencies), "crypto", "node:fs", "node:fs/promises", "node:path", "node:os"],
```

Comment:
this suppresses some rollup warning. Not 100% it is correct to fix it like this?

Reply:
Yes, that's what I've done with the other packages